app/assets/js/rtMedia.backbone.js

```diff
@@ -738,7 +738,15 @@ jQuery( function( $ ) {
 			uploaderObj.uploader.bind( 'UploadProgress', function( up, file ) {
 				//$("#" + file.id + " .plupload_file_status").html(file.percent + "%");
 				//$( "#" + file.id + " .plupload_file_status" ).html( rtmedia_uploading_msg + '( ' + file.percent + '% )' );
-				$( '#' + file.id + ' .plupload_file_status' ).html( '<div class="plupload_file_progress ui-widget-header" style="width: ' + file.percent + '%;"></div>' );
+				// creates a progress bar to display file upload status
+				var progressBar = jQuery( '<div/>', {
+					'class': 'plupload_file_progress ui-widget-header',
+				});
+				progressBar.css( 'width', file.percent + '%');
```

Comment:
`progressBar.css( 'width', file.percent + '%' );`

Please make sure you follow coding standards.